crates/api_models/src/user.rs

```diff
@@ -149,13 +149,25 @@ pub struct UserDetails {
     pub last_modified_at: time::PrimitiveDateTime,
 }
 
+#[derive(serde::Serialize, Debug, Clone)]
+pub struct GetUserDetailsResponse {
+    pub merchant_id: String,
+    pub name: Secret<String>,
+    pub email: pii::Email,
+    pub verification_days_left: Option<i64>,
+    pub role_id: String,
+    //this field is added for audit/debug reasons
```

Comment:
I know that we use this comment every where but from now let's correct it.
```suggestion
    // This field is added for audit/debug reasons
```